schema/me/update_conversation.js

```diff
@@ -15,20 +16,27 @@ export default mutationWithClientMutationId({
     buyer_outcome: {
       type: new GraphQLNonNull(BuyerOutcomeTypes),
     },
-    id: {
-      type: new GraphQLNonNull(GraphQLString),
+    ids: {
+      type: new GraphQLList(GraphQLString),
     },
   },
-  outputFields: ConversationFields,
-  mutateAndGetPayload: ({ buyer_outcome, id }, request, { rootValue: { accessToken } }) => {
+  outputFields: {
+    conversations: {
+      type: new GraphQLList(ConversationType),
+      resolve: (conversations) => conversations,
```

Comment:
Yea, in this case there is no field called `conversations`. Instead I'm resolving the entire body as `conversations`.